### ISSUE TITLE
[Enhancement] release Releases on Repository of Chocolatey packages

### ISSUE THREAD

**Issue by 4-FLOSS-Free-Libre-Open-Source-Software** [JanisEst/KeePassQRCodeView#11]
Hello, this is not an issue in itself. Just an suggestion. Would be nice to have a auto update feature for this primary security applications plugin from trusted source. The sha256 hash of PLRGX can be verified by installation automatically optionally if specified.  _For the lazy ones :) and_ to fix incompatibility issue with future of keepass version updates with plugin by getting all in one place for example.

I don't know if you have the spare time to maintain in parallel the "packages". Which basically would contains a script to URL of github [latest](https://github.com/JanisEst/KeePassQRCodeView/releases/latest) release asset. 

_If you may want to take a look here if you don't know already:
[keepass plugins packages](https://chocolatey.org/packages?q=keepass&moderatorQueue=&prerelease=true&sortOrder=package-download-count)
(There are many plugins and packages count is growing. )_
